cime_config/buildnml

```diff
@@ -389,7 +402,7 @@ def _create_runseq(case, coupling_times, valid_comps):
                 from runseq_TG import gen_runseq
             elif (comp_atm == 'ufsatm' and comp_ocn == "mom" and comp_ice == 'cice'):
                 from runseq_NEMS import gen_runseq
-            else: 
+            else:
                 from runseq_general import gen_runseq
```

Comment:
Not for this PR but it might be nice to have xmlchnage option to bypass using aoflux calculation  without introducing new custom Python function to generate run sequence.

Reply:
Good point. Can you please raise this as an enhancement issue.

Reply:
It is linked to 
https://github.com/ESCOMP/CMEPS/issues/76